integrationtests/src/test/java/io/kroxylicious/proxy/KroxyConfigBuilder.java

```diff
@@ -34,10 +34,19 @@ public record Cluster(@JsonGetter("bootstrap_servers") String bootstrapServers)
     public record Filter(String type, @JsonInclude(NON_EMPTY) Map<String, Object> config) {
     }
 
+    public record AdminHttp(Endpoints endpoints) {
+    }
+
+    public record Endpoints(@JsonInclude(NON_NULL) Map<String, String> prometheus) {
```

Comment:
nit: variable name, should be something like endpoints rather than `prometheus`

Reply:
The `Map<String, String>` is the empty object with the key `prometheus`.

Like:
```
endpoints:
  prometheus: {}
```

I can see why it reads a bit awkwardly though. Maybe it should be `prometheusEndpointConfig` and get mapped to `prometheus` by a jackson annotation.

Reply:
Urgh Ok. That's a bit crufty but ok.